src/thunks/fetchNotes.js

```diff
@@ -0,0 +1,19 @@
+import { setLoading, setError, setNotes } from '../actions'
+
+export const fetchNotes = () => {
```

Comment:
I would love to go through this together so I can get a better understanding of how the thunk works.